src/PackageDependencyViewExtension/PackageDependencyView.xaml

```diff
@@ -26,7 +26,7 @@
             RowBackground="Transparent"
             FontSize="14"
             IsHitTestVisible="False"
-            Grid.Row="0">
+            Grid.Row="1">
```

Comment:
Row number always start from 0 right? Why did we make this change?

Reply:
If we have to can we all start from row 0

Reply:
@QilongTang  We need to discuss this. All the comments except this have been addressed.

Reply:
Can we put both of them in Row 0? Currently Row 0 is empty correct from my understanding.

Reply:
Yes, we can do that. As long as both the grid's are set to same row value, it is fine. Changed both to value 0.